README.md

```diff
@@ -823,8 +823,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
-      `A string containing the ${nextNumber}.`;
+      return `A string containing the ${number}.`;
```

Comment:
wrong examples are bad examples, and bad examples are wrong examples. This is explicitly here because implicit return is a new feature in JS, and many people have misunderstood it.